### PR TITLE
Remove preheader for email confirmation message

### PR DESCRIPTION
**Pull request by kirill-sizov**
### Motivation and context
Sometimes a see this in my emails:

Screenshot from 2023-07-19 10-42-44

### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into the [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

**Comment by Marishka17** [comment]
Probably, we need to correct the actual text itself, and not delete the preheader.

**Comment by kirill-sizov** [comment]
> Probably, we need to correct the actual text itself, and not delete the preheader.

I think we don't need it. If the preheader is empty then email managers will show it as the first sentence of the letter (as it is already done for the welcome letter):
Screenshot from 2023-07-14 18-15-47

File: cvat/apps/iam/templates/account/email/email_confirmation_message.html [hunk 1]
```diff
@@ -107,12 +107,6 @@
 </head>
 <body style="background-color: #e9ecef;">
 
-  <!-- start preheader -->
-  <div class="preheader" style="display: none; max-width: 0; max-height: 0; overflow: hidden; font-size: 1px; line-height: 1px; color: #fff; opacity: 0;">
-    A preheader is the short summary text that follows the subject line when an email is viewed in the inbox.
```
Comment (nmanovic) [inline cvat/apps/iam/templates/account/email/email_confirmation_message.html:112]: Please insert something like: "Get Started with CVAT: Watch Product Tour, Explore Documentation, and Enjoy Exclusive Offer - Your Smooth Start Awaits!"

**Comment by nmanovic** [comment]
@kirill-sizov , great catch! I believe first line of our mail is useless as a preheader. Let's use something more informative.

File: cvat/apps/iam/templates/account/email/email_confirmation_message.html (hunk 1 above)
Comment (kirill-sizov) [reply cvat/apps/iam/templates/account/email/email_confirmation_message.html:112]: Yes, I agree probably we need to have a nice-looking preheader for the Welcome letter and I have added your suggestion in a separate PR.

But this PR removes the preheader for the email confirmation letter and users will see the following sentence "Confirm Your Email Address Thank you for signing up for CVAT!"

**Review by nmanovic — APPROVED** [review]

**Pull request merged**